Remove duplicate percent sign from MetaMask fee

Gudahtt opened the pull request:
The MetaMask fee is shown with two percent signs on the view quote page, because the percent sign is embedded in the fee amount as well as in the localized message.

The fee amount used now comes from the API, and does not have a percent sign. The percent sign is now only in the localized message. This allows for different locales to display the percentage differently. The old hard-coded value with a percent sign embedded has been removed, as it is no longer used anywhere.

Gudahtt committed:
Remove duplicate percent sign from MetaMask fee

The MetaMask fee is shown with two percent signs on the view quote
page, because the percent sign is embedded in the fee amount as well as
in the localized message.

The percent sign has been removed from the amount, and is now only in
the localized message. This allows for different locales to display the
percentage differently.

Note that we should still do further deduplication, as the fee is
currently hard-coded, fetched from the `/fee` endpoint, and embedded in
each individual quote. Currently the fee embedded in the quote is not
used, but the other two are.

Gudahtt commented:
<details>
<summary>Screenshots:</summary>
Before:

![double-percent](https://user-images.githubusercontent.com/2459287/96519390-77fd5080-1247-11eb-94c0-26c3c1bba3d9.png)

After:

![percent-after](https://user-images.githubusercontent.com/2459287/96519490-ada23980-1247-11eb-87d0-7668e464e7d4.png)

</details>

danjm commented:
This was my mistake. I think we should just use the fee from the api. We could use `const metaMaskFee = useSelector(getMetaMaskFeeAmount)` in this file instead of getting it from `renderableDataForUsedQuote`

danjm commented:
My above comment does not need to block this PR or release, but we will want to switch to the api value only before release to chrome store.

Gudahtt committed:
Remove the hard-coded fee; use API fee instead

The hard-coded MetaMask fee has been removed. The spot where this was
used has been replaced with the fee in the swaps Redux slice, which is
from the API `/fee` route.

Gudahtt commented:
@danjm I have updated this to remove the hard-coded fee, and use the API fee instead. It was a simple enough change.
